### PR TITLE
Translate doc node-conformance into chinese

### PR DESCRIPTION
**Pull request by lichuqiang**
chinese doc translation

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5564)

### CODE REVIEW

File: cn/docs/admin/node-conformance.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+## 限制
+
+在Kubernetes1.5版本中, 节点一致性测试存在以下限制:
+
+* 节点一致性测试只支持Docker作为容器运行时环境。
+
+## 节点的前提条件
+
+为运行节点一致性测试, 节点必须满足与标准的Kubernetes节点同样的前提条件. 最基本地, 节点应安装以下组件:
```
Comment (xiangpengzhao) [inline cn/docs/admin/node-conformance.md]: 中文语句中的英文标点需要统一修改为中文标点。

File: cn/docs/admin/node-conformance.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+# $CONFIG_DIR 是Kubelet的manifest文件路径。
+# $LOG_DIR 是测试结果输出的路径。
+sudo docker run -it --rm --privileged --net=host \
+  -v /:/rootfs -v $CONFIG_DIR:$CONFIG_DIR -v $LOG_DIR:/var/result \
+  gcr.io/google_containers/node-test:0.2
+```
+
+## 针对其他系统结构运行节点一致性测试
+
+Kubernetes 也为其他结构的系统提供了节点一致性测试的Docker镜像:
```
Comment (xiangpengzhao) [inline cn/docs/admin/node-conformance.md]: 结构-->架构？

File: cn/docs/admin/node-conformance.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+  Arch  |       Image       |
+--------|:-----------------:|
+ amd64  |  node-test-amd64  |
+  arm   |    node-test-arm  |
+ arm64  |  node-test-arm64  |
+
+## 运行选定的测试
+
+为运行指定的测试, 以你需要运行的测试的正则表达式来覆盖 `FOCUS` 环境变量。
```
Comment (xiangpengzhao) [inline cn/docs/admin/node-conformance.md]: 以-->用？下同

File: cn/docs/admin/node-conformance.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+sudo docker run -it --rm --privileged --net=host \
+  -v /:/rootfs:ro -v $CONFIG_DIR:$CONFIG_DIR -v $LOG_DIR:/var/result \
+  -e SKIP=MirrorPod \ # 运行除MirrorPod外的所有测试
+  gcr.io/google_containers/node-test:0.2
+```
+
+节点一致性测试是 [node e2e test](https://github.com/kubernetes/community/blob/{{page.githubbranch}}/contributors/devel/e2e-node-tests.md)的一个容器化的版本.
+默认情况下, 它会运行所有的一致性测试用例.
+
+理论上讲, 只要你合理地配置容器和挂载所需的卷，你可以运行任何的节点 e2e 测试用例。 但是这里 **强烈建议只运行一致性测试**, 因为运行非一致性测试需要很多复杂的配置.
```
Comment (xiangpengzhao) [inline cn/docs/admin/node-conformance.md]: ”讲“可以去掉。

**Comment by lichuqiang** [comment]
@xiangpengzhao 已更新，感谢

**Comment by zhangxiaoyu-zidif** [comment]
@xiangpengzhao , this PR has been fixed, how about it now to you?

**Comment by steveperry-53** [comment]
@xiangpengzhao Have all your issues been resolved? Are you ready to give this an LGTM? Thanks.

**Comment by xiangpengzhao** [comment]
I'm on vocation, so sorry for the delay. There are still some punctuations such as comma should be in Chinese format. And maybe some wording can be better. I'd like @tengqm to take another look.

**Comment by lichuqiang** [comment]
@tengqm could you plz help take a look at the PR?

File: cn/docs/admin/node-conformance.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+为运行指定的测试，用需要运行的测试的正则表达式来覆盖 `FOCUS` 环境变量。
+
+```shell
+sudo docker run -it --rm --privileged --net=host \
+  -v /:/rootfs:ro -v $CONFIG_DIR:$CONFIG_DIR -v $LOG_DIR:/var/result \
+  -e FOCUS=MirrorPod \ # 只运行MirrorPod测试
+  gcr.io/google_containers/node-test:0.2
+```
+
+为跳过指定的测试，用需要跳过的测试的正则表达式来覆盖 `SKIP` 环境变量。
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/admin/node-conformance.md]: 为跳过指定的测试，用需要跳过的测试的正则表达式来覆盖 `SKIP` 环境变量。
->
使用`SKIP`搭配您不想测试的测试用例，就可以在测试时跳过它们。

**Comment by zhangxiaoyu-zidif** [comment]
ping @tengqm  and @xiangpengzhao

**Comment by xiangpengzhao** [comment]
Ah, I think the doc author @Random-Liu is the best reviewer for this PR. I believe he can do both tech review and CN review :v:

**Comment by zhangxiaoyu-zidif** [comment]
ping @tengqm 
could you have some time to review this PR?

File: cn/docs/admin/node-conformance.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+- Random-Liu
+title: 验证节点设置
+---
+
+* TOC
+{:toc}
+
+## 节点一致性测试
+
+*节点一致性测试* 是一个容器化的测试框架，它提供了针对节点的操作系统验证和功能测试。该测试主要检测节点是否满足Kubernetes的最低要求; 通过检测的节点有资格加入Kubernetes集群。
```
Comment (tengqm) [inline cn/docs/admin/node-conformance.md]: system verification -> 系统验证
不要译成“操作系统验证"

File: cn/docs/admin/node-conformance.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
+---
+approvers:
+- Random-Liu
+title: 验证节点设置
+---
+
+* TOC
+{:toc}
+
+## 节点一致性测试
```
Comment (tengqm) [inline cn/docs/admin/node-conformance.md]: conformance -> 合规性或符合性
一致性一般指英文的consistency、uniformity，所以译成一致性很容易引起误解。

File: cn/docs/admin/node-conformance.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+## 节点的前提条件
+
+为运行节点一致性测试，节点必须满足与标准的Kubernetes节点同样的前提条件。 最基本地，节点应安装以下组件:
+
+* 容器运行时 (Docker)
+* Kubelet
+
+## 运行节点一致性测试
+
+为运行节点一致性测试，执行以下步骤:
```
Comment (tengqm) [inline cn/docs/admin/node-conformance.md]: 这句或者添加主语，或者调整语序

File: cn/docs/admin/node-conformance.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+1. 将 Kubelet定向到本机 `--api-servers="http://localhost:8080"`，
```
Comment (tengqm) [inline cn/docs/admin/node-conformance.md]: 令kubelet指向本机

File: cn/docs/admin/node-conformance.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+因为测试框架会启动一个本地的master来测试Kubelet。还有一些其他Kubelet参数可能需要关心:
```
Comment (tengqm) [inline cn/docs/admin/node-conformance.md]: 把”因为..."提前，更符合汉语习惯。

Comment (tengqm) [reply cn/docs/admin/node-conformance.md]: 关心->注意

File: cn/docs/admin/node-conformance.md [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+  * `--pod-cidr`: 如果使用 `kubenet`，需要为Kubelet任意指定一个 CIDR，例如 `--pod-cidr=10.180.0.0/24`。
+  * `--cloud-provider`: 如果使用了 `--cloud-provider=gce`，需要移除这个标记来运行测试。
```
Comment (tengqm) [inline cn/docs/admin/node-conformance.md]: 标记-> 参数

File: cn/docs/admin/node-conformance.md [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+
+```shell
+# $CONFIG_DIR 是Kubelet的manifest文件路径。
+# $LOG_DIR 是测试结果输出的路径。
+sudo docker run -it --rm --privileged --net=host \
+  -v /:/rootfs -v $CONFIG_DIR:$CONFIG_DIR -v $LOG_DIR:/var/result \
+  gcr.io/google_containers/node-test:0.2
+```
+
+## 针对其他架构运行节点一致性测试
```
Comment (tengqm) [inline cn/docs/admin/node-conformance.md]: architecture在这个上下文中指的是硬件体系结构，而不是一般意义的software architecture，所以应该翻译成“硬件体系结构”

File: cn/docs/admin/node-conformance.md [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+
+  Arch  |       Image       |
+--------|:-----------------:|
+ amd64  |  node-test-amd64  |
+  arm   |    node-test-arm  |
+ arm64  |  node-test-arm64  |
+
+## 运行选定的测试
+
+为运行指定的测试，用需要运行的测试的正则表达式来覆盖 `FOCUS` 环境变量。
```
Comment (tengqm) [inline cn/docs/admin/node-conformance.md]: 用需要运行的测试的正则表达式来覆盖 `FOCUS` 环境变量。
->
用正则表达式来描述将要运行的测试，并重载`FOCUS`环境变量。

File: cn/docs/admin/node-conformance.md [hunk 14]
```diff
@@ -0,0 +1,84 @@
...
+为跳过指定的测试，用需要跳过的测试的正则表达式来覆盖 `SKIP` 环境变量。
+
+```shell
+sudo docker run -it --rm --privileged --net=host \
+  -v /:/rootfs:ro -v $CONFIG_DIR:$CONFIG_DIR -v $LOG_DIR:/var/result \
+  -e SKIP=MirrorPod \ # 运行除MirrorPod外的所有测试
+  gcr.io/google_containers/node-test:0.2
+```
+
+节点一致性测试是 [节点 e2e 测试](https://github.com/kubernetes/community/blob/{{page.githubbranch}}/contributors/devel/e2e-node-tests.md)的一个容器化的版本。
```
Comment (tengqm) [inline cn/docs/admin/node-conformance.md]: 节点e2e测试
->
节点端到端测试

**Comment by lichuqiang** [comment]
@tengqm 已修改

**Comment by zhangxiaoyu-zidif** [comment]
ping @xiangpengzhao 
This PR could be OK, merge it.

**Comment by xiangpengzhao** [comment]
This LGTM. Thanks!

**Pull request merged**